test/unit/spec/ForgotPassword_spec.js

```diff
@@ -96,9 +96,9 @@ Expect.describe('ForgotPassword', function () {
         expect($usernameLabel.text().trim()).toEqual('Email or Username');
       });
     });
-    itp('does not allow autocomplete', function () {
+    itp('sets autocomplete for username', function () {
       return setup().then(function (test) {
-        expect(test.form.getUsernameAutocomplete()).toBe('off');
+        expect(test.form.getUsernameAutocomplete()).toBe('username');
```

Comment:
Is there any security concern in enabling autocomplete for username when in the Forgot Password scenario? Asking since this specifically checked for the autocomplete to be off.

Reply:
@haishengwu-okta ^

Reply:
Confirmed it's ok to enable autocomplete on case by case basis, including this scenario.